renovate.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "github>nginxinc/k8s-common"
+  ],
+  "kubernetes": {
+    "fileMatch": [
+      "examples/.+\\.yaml$",
+      "suite/.+\\.yaml$"
```

Comment:
Does this make it so renovate ignores these files?

Reply:
The opposite, they don't enable it by default because there's not an easy way to identify k8s files and they want to avoid parsing all the yamls by default

Reply:
Do we have anything in there that would need updating?